netbox/extras/management/templates/generated_schema.json.j2

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
If `TEMPLATES_DIR` is used, then the file should be moved to the common `templates` directory

Reply:
Absolutely agree, this is being implemented

Reply:
This is implemented now.